### PR TITLE
[13.0][MIG] maintenance_account: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0 (from 14.0).

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT41743

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

File: maintenance_account/models/account_move.py [hunk 1]
```diff
@@ -35,6 +35,12 @@ def action_post(self):
                 vals = line._prepare_equipment_vals()
                 equipment_ids = []
                 for _i in range(1, limit):
+                    # Prevent error related to dupplicate followers
+                    equipment_model = (
+                        equipment_model.with_context(mail_create_nosubscribe=True)
```
Comment (pedrobaeza) [inline maintenance_account/models/account_move.py]: We need the subscribers, so you should find another method.

File: maintenance_account/models/account_move.py [hunk 2]
```diff
@@ -32,10 +32,9 @@ def action_post(self):
                     line._set_equipment_category()
                 # Create equipments
                 limit = int(line.quantity) + 1
-                vals = line._prepare_equipment_vals()
                 equipment_ids = []
                 for _i in range(1, limit):
-                    equipment = equipment_model.create(vals)
+                    equipment = equipment_model.create(line._prepare_equipment_vals())
```
Comment (pedrobaeza) [inline maintenance_account/models/account_move.py]: Instead of this, do a copy of the vals.

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**